### PR TITLE
CON-1594: Added troubleshooting section for 1password

### PR DESCRIPTION
**Pull request by keyur-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by keyur-sumo** [comment]
@kimsauce @nmohantyctr @stacykor : Can you please help with review?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 1]
```diff
@@ -341,3 +341,24 @@ The following table shows the **config** parameters for a 1Password Source.
...
+The following section details how you can resolve various errors: 
+
+**Error Code:**  401 <br />
+**Error Details:**  {"Error":{"Message":"Unauthorized"}}
+
+To resolve this:
+
+- Make sure Base URL is as per your domain.
+- Make sure correct API Token is used to configure the source.
+- If you still see the 401 Unauthorized error in the Status column, then regenerate the API Token by following [Authentication](#authentication) steps, and update the API Token for the source.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:364]:
```suggestion
After configuring your Source, you should check the status of the source in the **Collectors** page >  **Status** column. If the Source is not functioning as expected, you may see an error next to the Source Category column as shown below: 

![troubleshooting.jpg](/img/send-data/1password-troubleshooting.jpg)

**Error Code**:  `401` <br />
**Error Details**:  `{"Error":{"Message":"Unauthorized"}}`

To resolve these errors:
- Make sure the Base URL matches your domain.
- Make sure correct API Token is used to configure the source.
- If you're still seeing the `401 Unauthorized error` in the **Status** column, then regenerate the API Token by following [Authentication steps](#authentication), and update the API Token for the source.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Style edits

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 2]
```diff
@@ -341,3 +341,18 @@ The following table shows the **config** parameters for a 1Password Source.
...
+```
+
+## Troubleshooting
+
+After configuring your Source, you should check the status of the source in the **Collectors** page >  **Status** column. If the Source is not functioning as expected, you may see an error next to the Source Category column as shown below: 
+
+![troubleshooting.jpg](/img/send-data/1password-troubleshooting.jpg)
+
+**Error Code**:  `401` <br />
+**Error Details**:  `{"Error":{"Message":"Unauthorized"}}`
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:353]:
```suggestion
**Error Code**: `401` <br />
**Error Details**: `{"Error":{"Message":"Unauthorized"}}`
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 3]
```diff
@@ -341,3 +341,18 @@ The following table shows the **config** parameters for a 1Password Source.
...
+
+![troubleshooting.jpg](/img/send-data/1password-troubleshooting.jpg)
+
+**Error Code**: `401` <br />
+**Error Details**: `{"Error":{"Message":"Unauthorized"}}`
+
+To resolve these errors:
+- Make sure the Base URL matches your domain.
+- Make sure correct API Token is used to configure the source.
+- If you're still seeing the `401 Unauthorized error` in the **Status** column, then regenerate the API Token by following [Authentication steps](#authentication), and update the API Token for the source.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:358]:
```suggestion
- If you're still seeing the `401 Unauthorized error` in the **Status** column, regenerate the API Token by following [these Authentication steps](#authentication) and then updating the API Token for the source.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**